ursina/prefabs/text_field.py

```diff
@@ -267,7 +267,7 @@ def get_mouse_position(self):
 
 
     def input(self, key):
-        print('-------------', key)
+        #print('-------------', key)
```

Comment:
Coding style: add a space before the comment text
`# print('-------------', key)`